cardano-sl-src.json

```diff
@@ -0,0 +1,6 @@
+{
+    "url": "https://github.com/input-output-hk/cardano-sl",
+    "fetchSubmodules": "true",
+    "rev": "1bba2fd0183f575752f4529182205e04074db336",
+    "sha256": "1qfi307x7nfp42ldb6sjx30mk7b0zfvd4b48pmyd9g465f7sp420"
+}
```

Comment:
I wonder if we want to (eventually, not now!) have a Github repo that defines versions in a central manner -- so we don't have these source specs littered all around..

Reply:
Why would you need to do that?

Reply:
To establish a single source of truth, and ensure that all of our builds get the same inputs.

In the end it will translate to higher Hydra cache hit rate and less confusion.

Reply:
In practice that won't work - we had situations where we deployed different version of cardano-sl to users than it's running in the network. Just to allow shipping hotfixes faster.